src/utils/network.ts

```diff
@@ -251,7 +251,13 @@ export const loginRequest = (
     };
 
     if (isVerbose) {
-        logInfo(`\nRequesting ${config.loginUrl}...\n\n`);
+        logInfo(`\nRequesting ${config.loginUrl}...`);
+        logInfo(
+            `with data: ${JSON.stringify(data).replace(
```

Comment:
Or do `delete data.password` before calling JSON.stringify.

Reply:
I removed the regex, thanks.